IncrementalSVDFastUpdate.C

```diff
@@ -171,6 +174,44 @@ IncrementalSVDFastUpdate::computeBasis()
      delete d_basis_right;
      d_basis_right = new Matrix(*d_W);
    }
+
+  if(d_rank == 0) {
+       std::cout << "d_num_samples = " << d_num_samples << "\n";
+       std::cout << "d_num_rows_of_W = " << d_num_rows_of_W << "\n";
+       std::cout << "d_singular_value_tol = " << d_singular_value_tol << "\n";
+       std::cout << "smallest SV = " << d_S->item(d_num_samples-1,d_num_samples-1) << "\n";
+       std::cout << "next smallest SV = " << d_S->item(d_num_samples-2,d_num_samples-2) << "\n";
+   }
+   // remove the smallest singular value if it is smaller than d_singular_value_tol
```

Comment:
I have a question. Is computeBasis called multiple times, at every iteration of the IncrementalSVD? Or is it only called once by buildInitialSVD?

Reply:
computeBasis is also called in "buildIncrementalSVD" in IncrementalSVD.C. It is supposed to be called whenever the incremental basis is updated.